components/molecules/Card.js

```diff
@@ -49,7 +49,9 @@ export default function Card(props) {
                             {props.post.primary_tag.name}
                         </a>
                     </Link>
-                    <h2 className={styles.title}><a href="#" className={styles.cardtitle}>{props.post.title}</a></h2>
+                    <h2 className={styles.title}>
+                        <a href={props.post.url} className={styles.cardtitle}>{props.post.title}</a>
```

Comment:
Just tested it, but the `props.post.url` return the url with ghost website as host (_busrides.ghost.io_)!
From the `Link `at the top, I think It should be `/${props.t.getLocale}/${props.post.slug}` instead:
```suggestion
                        <a href={`/${props.t.getLocale}/${props.post.slug}`} className={styles.cardtitle}>{props.post.title}</a>
```

Reply:
hadn't noticed the ghost thing: clicked, got me to the right page, magic!
making fix on both card and compact card (don't know the difference though)